src/elements/icon/demo/icons.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html ng-app="semantic.ui.angular.icons">
+<head lang="en">
+    <meta charset="UTF-8">
+    <title>Semantic-UI Demo page</title>
+    <link rel="stylesheet" href="../../../../docs/config/assets/semantic-ui/semantic.css"/>
+</head>
+<body style="padding:100px;text-align:center;">
+
+<sm-icon sm-font-icon="child icon" ng-style="{ color: 'red' }"></sm-icon>
```

Comment:
I was wondering if we could add more features later like svg Icons, hence added the directive.